### PR TITLE
Refactor convert into a separate package 

### PR DESCRIPTION
**Pull request by ZhiminXiang**

### CODE REVIEW

File: k8s.go [hunk 1]
```diff
@@ -18,6 +18,8 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	"k8s.io/client-go/tools/clientcmd"
 	"sigs.k8s.io/yaml"
+
+	convertpkg "github.com/istio-ecosystem/security-policy-migrate/converter"
```
Comment (yangminzhu) [inline k8s.go]: Better to modify the variable `converter` instead of renaming the imported package.

File: util.go [hunk 2]
```diff
@@ -2,26 +2,16 @@ package main
...
-type objectStruct struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              map[string]interface{} `json:"spec,omitempty"`
-}
+	convertpkg "github.com/istio-ecosystem/security-policy-migrate/converter"
+)
 
-func convertToPolicy(item unstructured.Unstructured) (*InputPolicy, error) {
+func convertToPolicy(item unstructured.Unstructured) (*convertpkg.InputPolicy, error) {
```
Comment (yangminzhu) [inline util.go]: why not just move this whole file to converter/util.go?

File: k8s.go (hunk 1 above)
Comment (ZhiminXiang) [reply k8s.go]: Done

File: util.go (hunk 2 above)
Comment (ZhiminXiang) [reply util.go]: Done

**Pull request merged**